res/values-hu/strings.xml

```diff
@@ -1183,7 +1183,6 @@ Commands:
     <string name="pm">PM</string>
     <string name="You_are_already_friends_">You are already friends.</string>
     <string name="Account_must_be_1_day_old_to_send_mail_">Account must be 1 day old to send mail.</string>
-    <string name="Translator">Translator</string>
```

Comment:
I just think the "Translator" string shouldn't be removed. It belongs to one of the most current strings that is about to appear in the next update, so it should be translated or if you aren't about to change, keep it instead